src/components/GeneralInfo/GeneralInformation/GeneralInformation.js

```diff
@@ -23,219 +23,210 @@ import useInsightsNavigate from '@redhat-cloud-services/frontend-components-util
 import './general-information.scss';
 import { ConversionAlert } from './ConversionAlert';
 
-class GeneralInformation extends Component {
-  state = {
-    isModalOpen: false,
-    modalTitle: '',
-    modalVariant: 'small',
-  };
+const GeneralInformation = ({
+  store,
+  writePermissions,
+  SystemCardWrapper = SystemCard,
+  OperatingSystemCardWrapper = OperatingSystemCard,
+  BiosCardWrapper = BiosCard,
+  BootcImageCardWrapper = BootcImageCard,
+  InfrastructureCardWrapper = InfrastructureCard,
+  ConfigurationCardWrapper = ConfigurationCard,
+  SystemStatusCardWrapper = SystemStatusCard,
+  DataCollectorsCardWrapper = DataCollectorsCard,
+  CollectionCardWrapper = false,
+  SubscriptionCardWrapper = SubscriptionCard,
+  children,
+  navigate,
+  entity,
+  inventoryId,
+  loadSystemDetail,
+  systemProfilePrefetched = false,
+  showImageDetails = false,
+  isBootcHost = false,
+  showRuntimesProcesses = false,
+}) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [modalTitle, setModalTitle] = useState('');
+  const [modalVariant, setModalVariant] = useState('small');
```

Comment:
nit-pick:

```suggestion
  import { Grid, GridItem, Modal, ModalVariant } from '@patternfly/react-core';
  ...
  const [modalVariant, setModalVariant] = useState(ModalVariant.small);
```

so we take advantage of some constants available from PF/react